Code/Engine/Foundation/Time/Implementation/Clock_inl.h

```diff
@@ -59,18 +59,18 @@ inline double ezClock::GetSpeed() const
   return m_fSpeed;
 }
 
-inline void ezClock::SetMinimumTimeStep(ezTime tMin)
+inline void ezClock::SetMinimumTimeStep(ezTime min)
```

Comment:
't' prefix for ezTime?

Reply:
Unsure about this one. I often use t for local time variables, but not sure we should make this a requirement.